mne/annotations.py

```diff
@@ -567,6 +567,54 @@ def crop(self, tmin=None, tmax=None, emit_warning=False, verbose=None):
 
         return self
 
+    @verbose
+    def set_durations(self, mapping, verbose=None):
+        """Set annotation durations(s). Operates inplace.
+
+        Parameters
+        ----------
+        mapping : dict | float | int
+            A dictionary mapping the annotation description to a duration
+            e.g. {‘ShortStimulus’ : 3, ‘LongStimulus’ : 12}.
```

Comment:
```suggestion
            e.g. ``{'ShortStimulus' : 3, 'LongStimulus' : 12}``.
```
`` are optional but will make it look like code in the rendered doc.